src/fauxton/install.rst

```diff
@@ -12,66 +12,44 @@
 
 .. _fauxton/install:
 
-============
-Installation
-============
-
-Recent versions of `node.js`_ and `npm`_ are required.
-
-.. _node.js: http://nodejs.org/
-.. _npm: https://npmjs.org/doc/README.html
-
-Get the source
-==============
-
-Clone the CouchDB repo::
-
-    $ git clone http://git-wip-us.apache.org/repos/asf/couchdb.git
-    $ cd couchdb
-
+=============
 Fauxton Setup
 =============
 
-Install all dependencies::
+Fauxton is included with CouchDB 2.0, so make sure CouchDB is running, then go to::
 
-    couchdb/ $ cd src/fauxton
-    couchdb/src/fauxton/ $ npm install
+    http://127.0.0.1:5984/_utils/
 
-.. note::
-    To avoid a npm global install add ``node_modules/.bin`` to your path::
+You can also upgrade to the latest version of Fauxton by using npm::
 
-        export PATH=./node_modules/.bin:$PATH
+    $ npm install -g fauxton
+    $ fauxton
 
-    Or just use the wrappers in ``./bin/``.
+(Recent versions of `node.js`_ and `npm`_ are required.)
 
-    Development mode, non minified files::
-
-        ./bin/grunt couchdebug
-
-    Or fully compiled install::
-
-        ./bin/grunt couchdb
-
-Dev Server
-==========
+.. _node.js: http://nodejs.org/
+.. _npm: https://npmjs.org/doc/README.html
 
-Using the dev server is the easiest way to use Fauxton, specially when
-developing for it::
+Fauxton Visual Guide
+====================
+You can find the Visual Guide here:
+    http://couchdb.apache.org/fauxton-visual-guide
 
-    grunt dev
+Development Server
+==================
 
-Deploy Fauxton
-==============
+Recent versions of `node.js`_ and `npm`_ are required.
 
-Deploy Fauxton to your local CouchDB instance:
+.. _node.js: http://nodejs.org/
+.. _npm: https://npmjs.org/doc/README.html
 
-    ./bin/grunt couchapp_deploy
+Using the dev server is the easiest way to use Fauxton, specially when developing for it::
 
-The Fauxton be available by `/fauxton/_design/fauxton/index.html
-<http://localhost:5984/fauxton/_design/fauxton/index.html>`_
+    $ git clone https://github.com/apache/couchdb-fauxton.git
+    $ npm run dev
```

Comment:
I think they would need to do an `npm install` as well. so `npm install && npm run dev`